zk-kyc/README.md

```diff
@@ -0,0 +1,85 @@
+# zk-KYC Demonstration Application
+
+This demo leverages ID.me api to generate a client authentication token. The token includes a nonce that is associated with the user's connected wallet address, employing principles from the [OpenPubkey: Augmenting OpenID Connect with User held Signing Keys](https://eprint.iacr.org/2023/296) paper. The JWT's integrity is then verified within the guest using ID.me's public RS256 signing [certificates](https://api.id.me/oidc/.well-known/jwks). Subsequently, the guest generates a cryptographic proof of the JWT's integrity and issues a receipt. This receipt, containing the SNARK, an obfuscated identifier, and the user's address, can be validated on the onchain for ERC721 token minting or other transactions, if valid.
```

Comment:
What are we enabling with this demo? Can we speak to that before we get into the jargon? 

I think these sentences are more appropriate for a ## Technical Architecture section than the top-of-page introduction.

Reply:
Added a short sentence to give a quick one-liner.